CHANGELOG.md

```diff
@@ -44,7 +44,16 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
```

Comment:
@alessio there is no pending section in this file for the next 0.41 release. Where do these entries go?

Reply:
yeah I'm about to open a PR for that

Reply:
@alexanderbez for now just don't add an entry. I think it's much easier to prep and finalize both changelog and release notes at release time. So we do the work only once. Wdyt?

Reply:
I agree. But the changelog entries need to be tracked somewhere, right? How does the release manager know what to add?

Reply:
There you go: https://github.com/cosmos/cosmos-sdk/milestone/40

Reply:
All merged PRs must be included in the release milestone